docs/cli.md

```diff
@@ -403,9 +403,10 @@ Getting balance is easy:
 ./bin/neo-go wallet nep17 balance -w /etc/neo-go/wallet.json -r http://localhost:20332
 ```
 
-By default, you'll get data for all tokens for the default wallet's
-address. You can select non-default address with `-a` flag and/or select token
-with `--token` flag (token hash or name can be used as parameter).
+By default, you'll get data for all tokens that are owned by all accounts
+stored in the given wallet. You can a particular address with `-a` flag and/or
```

Comment:
`You can specify a...`?

Reply:
I'm trying to understand how this happens, either I'm too slow at typing (and compensating for that), or too fast at reading (and not noticing), but I'm just omitting words from time to time. Luckily, we have a review process.

Reply:
Fixed.